admin/package-files/after-install.sh

```diff
@@ -0,0 +1,2 @@
+# https://git.fedorahosted.org/cgit/firewalld.git/tree/config/macros.firewalld
```

Comment:
That file defines a macro, which expands to the line that I have. In a fedora/centos package, you would just put `%firewalld_reload`, but
1. I wanted this to be cross-platform.
2. It requires the firewalld-filesystem to be installed.
I think more sense to just install the snippet.